### DISCUSSION TITLE
What are the timeline dates on minor and major releases?

### DISCUSSION THREAD

**Discussion by JushBJJ** [dogecoin/dogecoin#2268] [Q&A] [answered]
Are there release schedules for future versions like the current milestones 1.14.4 and 1.21? EST if possible 🙂

**Comment by rnicoll** [comment, marked as answer]
As we're open source and volunteer driven, we don't have a fixed capacity, which means we can't usefully estimate delivery dates. Basically sooner if we get more skilled reviewers and coders, later if not, so right now I'm trying to help more people get involved so we can scale the team.
